thunder/benchmarks/layers_for_inference_benchmark.py

```diff
@@ -199,9 +201,8 @@ def dequantize_to_dtype(tensor_fp4, tensor_sf, global_scale, dtype, device, bloc
     return out
 
 
-# TODO: Update this accordingly to the progress of nvfp4 kernel implementation.
-# An alternative is to use `_register_nvfuser_translator` of https://github.com/Lightning-AI/lightning-thunder/pull/2481
-# instead of updating this function itself.
+# NOTE: This custom op is registered with nvfuser translator in benchmark_inference.py
+# using _register_nvfuser_translator. See benchmark_inference._register_nvfp4_ops().
 @torch.library.custom_op("nvf_cutlass::f16a_nvfp4weight_scaled_mm", mutates_args=())
 def nvfuser_f16a_nvfp4weight_scaled_mm(
```

Comment:
This function does not have nvfuser translator yet, does it?

Reply:
yes, when I initially wrote this code, I expected nvfp4 gemm to be available as well as grouped gemm but it actually isn't. I removed this function.